doc/Learn/includes/getting-help.md

```diff
@@ -0,0 +1,11 @@
+---
+uid: Uno.Extensions.GettingHelp
+---
+
+<!-- markdownlint-disable MD041 -->
+
+> [!TIP]
+> Take a look at our [common issues list](xref:Uno.UI.CommonIssues) if you're having any troubles.
+>
+> If you are experiencing issues with Uno Platform, please visit our [GitHub Discussions](https://github.com/unoplatform/uno/discussions) or [Discord Server](https://www.platform.uno/discord) - where our engineering team and community will be able to help you.
+>
```

Comment:
@eriklimakc simply using uid: Uno.Development.GettingHelp from the uno repo does not work with the include here? instead of duplicating the getting help file here
https://github.com/unoplatform/uno/blob/master/doc/articles/includes/getting-help.md?plain=1

Reply:
@agneszitte apparently it doesn't work with `xref:...`. I tried both, from uno repo and also the duplicated one in extensions and neither of them worked:

![image](https://github.com/user-attachments/assets/f9b013dc-9655-4278-a91d-5574297c1db4)

![image](https://github.com/user-attachments/assets/3a0b524c-3b4f-46a0-bc1c-9b50952e841b)

Reply:
Thanks for the confirmation and the tests @eriklimakc ! Appreciated!